kafka-cruise-control-stop.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").
+# See License in the project root for license information.
+
+SIGNAL=${SIGNAL:-TERM}
+PIDS=$(ps ax | grep -i 'cruise' | grep -i 'control' | awk '{print $1}')
```

Comment:
Does the following command work? -- i.e. it also ensures that the full phrase `cruise-control` is present, it is a java application and not `grep` itself:

`ps ax | grep -i 'cruise-control' | grep java | grep -v grep | awk '{print $1}'`

Reply:
Yes, this works perfectly! I've tested the command and verified that it works. :)